mimoto-issuers-config.json

```diff
@@ -401,6 +401,31 @@
       "proxy_token_endpoint": "https://${mosipid.identity.esignet.host}/v1/esignet/oauth/v2/token",
       "qr_code_type": "OnlineSharing",
       "enabled": "true"
+    },
+    {
+      "credential_issuer": "Land",
+      "display": [
+        {
+          "name": "Land Registry",
+          "logo": {
+            "url": "https://mosip.github.io/inji-config/logos/agro-vertias-logo.png",
+            "alt_text": "land-logo"
+          },
+          "title": "Land Registry",
+          "description": "Download Land Registry Credential",
+          "language": "en"
+        }
+      ],
+      "protocol": "OpenId4VCI",
+      "client_id": "${mimoto.oidc.mock.partner.clientid}",
+      "client_alias": "mpartner-default-mimoto-mock-oidc",
```

Comment:
alias not matching to https://github.com/mosip/inji-config/blob/release-0.5.x/mimoto-issuers-config.json#L186